### ISSUE TITLE
Getting a ? on mobile PageSpeed score

### ISSUE THREAD

**Issue by rulloliver** [GoogleChrome/lighthouse#11385]
Hi, 

Our site is getting a ? mobile PageSpeed score. This issue arised a few days ago. Currently unable to pinpoint the issue from the site side, is there any possibility to assist? 

Please view site: https://tinyurl.com/yyfx7kgl

Best

**Comment by patrickhulce** [comment]
Thanks for filing @rulloliver! I can reproduce this [in PSI](https://developers.google.com/speed/pagespeed/insights/?url=https%3A%2F%2Fwww.scoro.com%2F) but no other environment. We'll have to look into it.

**Comment by connorjclark** [comment]
![image](https://user-images.githubusercontent.com/4071474/92295979-72e17d80-eef5-11ea-8ca0-845fa54934d3.png)

weird issue with the css protocol

**Comment by paulirish** [comment]
perhaps related to https://github.com/GoogleChrome/lighthouse/issues/10876

**Comment by connorjclark** [comment]
> perhaps related to #10876

I don't think so. I think the root cause is `SPEEDINDEX_OF_ZERO`, aka no screenshots, for some reason ... Nothing painted = whole page was too busy = no CSS parsed yet ???

https://sentry.io/organizations/google-lighthouse/issues/1454681293/?project=174697&query=is%3Aunresolved+No+style+sheet&statsPeriod=14d

![image](https://user-images.githubusercontent.com/4071474/95264316-b715b580-07f4-11eb-9b4f-b14f05542c36.png)

many (but not all) have a NO_TTI or NO_LCP error too..

I can't get this to happen anymore with the provided URL.

**Comment by brendankenny** [comment]
> > perhaps related to #10876
> 
> I don't think so. I think the root cause is whatever caused `SPEEDINDEX_OF_ZERO`

I believe I suggested that comment in the last bug bash. It was just based off the CSSUsage/"no style sheet with given id found" error in the screenshot.

> Nothing painted = whole page was too busy = no CSS parsed yet ???

We only add a style sheet to be queried when the protocol indicates [one has been added (`CSS.styleSheetAdded`)](https://github.com/GoogleChrome/lighthouse/blob/9e0e9dfb67a3dd38b611b02d1129b197da8a2b0c/lighthouse-core/gather/gatherers/css-usage.js#L25), not on network records. I would assume it would only generate the event/id after it has something that can be queried?

Lots of interesting weird things about these artifacts, though (here's a [live version of the report](https://googlechrome.github.io/lighthouse/viewer/?gist=01d1b4d255557d866a98b73461e1243a) from them)
- It looks like the screenshot in https://github.com/GoogleChrome/lighthouse/issues/11385#issuecomment-687528372 doesn't match the attached artifacts, because those artifacts produce successful `css-usage` and TTI audits, even though you still get a `SPEEDINDEX_OF_ZERO`.
- There are screenshots in the trace, they're just five screenshots (three after FCP) of all white. Hence the zero.
- One possible bug for us here: it's questionable whether `SPEEDINDEX_OF_ZERO` should use the "Chrome didn't collect any screenshots" string, because it did collect screenshots.
- That trace has an FCP (what was it?) and an LCP (which starts with the contents `"By clicking \"Accept All Cookies\", you agree to the storing of cookies on your d…"`), but where are they in the screenshots? Nothing appears to be painted.
- several `net::ERR_BLOCKED_BY_CLIENT.Inspector` errors in there (#10362) and a `document.write` error via `googleadservices` and a js error on attempting to reference `myLinkToDecorate`

I'm not sure there's much for us to do here without a test case, though. It may just be there were enough errors in the site/PSI combo to not paint anything screenshottable but it was still enough to trigger the FCP/LCP thresholds.

**Comment by rulloliver** [comment]
Issue resolved currently by itself but lasted for about a week. Haven't been able to replicate it since. Thank you for the feedback and help

**Comment by connorjclark** [comment]
> One possible bug for us here: it's questionable whether SPEEDINDEX_OF_ZERO should use the "Chrome didn't collect any screenshots" string, because it did collect screenshots.


https://github.com/GoogleChrome/lighthouse/blob/d9e84e1d71e04ddae4f53d8e013683d6ca4c17ce/lighthouse-core/computed/speedline.js#L45

should we just check for `undefined` here, and allow for `0` to continue without error or throw a unique error for it?
